### PR TITLE
OSSMDOC-208: fixing issue in SMCP example.

### PR DESCRIPTION
**Pull request by JStickler**
Fixing issue in Service Mesh SMCP CR that leads to failure/error.

Edited text can be found in this topic  -> https://deploy-preview-31958--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-custom-resources.html#ossm-cr-example_ossm-custom-resources-v2x

Eng review -rcernich
QE review - gbaufake
Peer review - missmesss

### CODE REVIEW

**Comment by JStickler** [comment]
@rcernich  Could you review these changes to the SMCP example file please?

**Review by rcernich — APPROVED** [review]

**Review by rcernich — APPROVED** [review]

**Comment by JStickler** [comment]
@tvieira, @yxun   @gbaufake   request a QE review please.

**Review by gbaufake — APPROVED** [review]

**Comment by gbaufake** [comment]
@JStickler just tested with the smcp described.

**Comment by JStickler** [comment]
@gbaufake Thank you!

**Comment by yxun** [comment]
HI @JStickler  , is that a change for 1.1 or 2.0 CR ?

**Comment by yxun** [comment]
@JStickler @rcernich In the CR example,  the policy is using mixer but telemetry is using istiod. Is that an expected example or not ?

File: modules/ossm-cr-example.adoc [hunk 1]
```diff
@@ -211,10 +211,6 @@ spec:
         config:
           env: {}
           envSecrets: {}
-        persistence:
-          storageClassName: ""
-          accessMode: ReadWriteOnce
-          capacity: 5Gi
         service:
```
Comment (rcernich) [inline modules/ossm-cr-example.adoc:214]: I think these are okay to leave, although we might want to have a valid value for `storageClassName`.  We should also add an `enabled: false` entry under persistence too.  I think there's an open issue where `capacity` does not get applied.

Comment (JStickler) [reply modules/ossm-cr-example.adoc:214]: @xyun This CR example is for 2.0
@rcernich   Do you have an example that includes the changes you mention?  Or do you want to open a OSSMDOC issue and get those updates in a different PR?  (That is, are we OK to merge these changes or should I make those changes now?)

**Comment by missmesss** [comment]
Hi Julie, docs changes LGTM.

**Review by otikhomi — APPROVED** [review]

**Pull request merged**